en/components/bulletgraph.md

```diff
@@ -150,22 +148,20 @@ The ranges are visual elements that highlight a specified range of values on a s
 
 ### Tick Marks
 The tick marks serve as a visual division of the scale into intervals in order to increase the readability of the bullet graph.
-- Major tick marks – The major tick marks are used as primary delimiters on the scale. The frequency they appear at, their extents and style can be controlled by setting the corresponding properties.
+- Major tick marks – The major tick marks are used as primary delimiters on the scale. The frequency they appear at, their extents and style can be controlled by setting their corresponding properties.
 - Minor tick marks – The minor tick marks represent helper tick marks, which might be used to additionally improve the readability of the scale and can be customized in a way similar to the major ones.
 
 ```html
 <igx-bullet-graph
     height="80px" width="400px"
     minimumValue=0 value=70 interval=10
     maximumValue=100 targetValue=90
-
     tickBrush="DodgerBlue"
     ticksPreTerminal=0
     ticksPostInitial=0
     tickStrokeThickness=2
     tickStartExtent=0.2
     tickEndExtent=0.075
-
     minorTickCount=4
```

Comment:
I added these blank lines to separate required code (height, min/max) from code that show a given feature. IMO, we should keep these blank lines so that it is easier to read code instead of having long block of code.

Reply:
I think that it looks strange. can we add comments instead?